src/app/zap-templates/zcl/data-model/chip/joint-fabric-pki-cluster.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="ICACSigningRequestStatusEnum" type="enum8">
+    <cluster code="0x0753"/>
+    <item name="OK" value="0x00"/>
+    <item name="InvalidIcaCsrFormat" value="0x01"/>
+    <item name="InvalidIcaCsrSignature" value="0x02"/>
+    <item name="FailedDclVendorIdValidation" value="0x03"/>
+    <item name="NotAnIcac" value="0x04"/>
+    <item name="BusyAnchorTransfer" value="0x05"/>
+    <item name="IcaCsrSigningFailed" value="0x06"/>
+    <item name="IcaCsrRequestNoUserConsent" value="0x07"/>
+  </enum>
```

Comment:
All the above error codes were removed in https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10743. We decided that the errors for ICACCSRRequest are conveyed by the generic IM errors that exists already and that there is no need to introduce JF specific error codes.